src/main/kotlin/com/team4099/robot2023/subsystems/drivetrain/swervemodule/SwerveModuleIOFalcon.kt

```diff
@@ -172,6 +274,29 @@ class SwerveModuleIOFalcon(
     inputs.driveTemp = driveFalcon.deviceTemp.value.celsius
     inputs.steeringTemp = steeringFalcon.deviceTemp.value.celsius
 
+    inputs.odometryDrivePositions =
+      drivePositionQueue
+      .stream()
+      .map { value: Double ->
+        (
+          DrivetrainConstants.WHEEL_DIAMETER * 2 * Math.PI * value /
+            DrivetrainConstants.DRIVE_SENSOR_GEAR_RATIO
+          )
+      }
+      .toArray() as
+      Array<Length>
+    inputs.odometrySteeringPositions =
+      steeringPositionQueue
+      .stream()
+      .map { value: Double ->
+        Rotation2d.fromRotations(value / DrivetrainConstants.STEERING_SENSOR_GEAR_RATIO)
```

Comment:
can just do `(value / DrivetrainConstants.STEERING_SENSOR_GEAR_RATIO).rotations` instead of creating a rotation2d object and then .radians it